mobile/fastlane/metadata/android/fr-FR/full_description.txt

```diff
@@ -0,0 +1,10 @@
+Cozy Cloud permet à chacun d’avoir son cloud personnel, respectueux de la vie privée. Avec Cozy Files, accédez aux fichiers et aux photos que vous stockez dans votre Cozy de partout.
```

Comment:
That's right.
What about: "Avec Cozy Files, accédez aux fichiers et aux photos stockés dans votre Cozy depuis n'importe quel appareil." ?